xml/Microsoft.Win32.SafeHandles/SafeNCryptHandle.xml

```diff
@@ -34,7 +34,24 @@
   </Attributes>
   <Docs>
     <summary>Provides a safe handle that can be used by Cryptography Next Generation (CNG) objects.</summary>
-    <remarks>To be added.</remarks>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+      
+## Remarks
+
+
+This class wraps up the logic to correctly duplicate and free these handles to simulate a native duplication.
+            
+Each open handle object can be thought of as being in one of three states:
+
+- **Owner** - Created via the marshaler, traditional style safe handle. Notably, only one owner handle exists for a given native handle.
+- **Duplicate** - Points at a handle in the Holder state. Releasing a handle in the duplicate state results only in decrementing the reference count of the holder, not in a release of the native handle.
+- **Holder** - Holds onto a native handle and is referenced by handles in the duplicate state.
+
+When all duplicate handles are closed, the holder handle releases the native handle. A holder handle will never be finalized, since this results in a race between the finalizers of the duplicate handles and the holder handle. Instead, it relies upon all of the duplicate handles to be finalized and decrement the ref count to zero. Instances of a holder handle should never be referenced by anything but a duplicate handle.
```

Comment:
Looking at [here](https://referencesource.microsoft.com/#System.Core/Microsoft/Win32/SafeHandles/NCryptSafeHandles.cs,a85c6ffc1b440ffa), I see this whole line is continued explanation for **Holder** shouldn't it be in the same line above?

Reply:
Nice catch, you're right. I'll fix it.